authors/apps/articles/views.py

```diff
@@ -29,6 +29,16 @@ def post(self, request, *args, **kwargs):
 
     def create(self, request, *args, **kwargs):
         payload = request.data.get('article', {})
+        the_tags = payload.get("tagList", None)
+        tags_pk = []
+        if the_tags:
+            the_tags = payload.pop("tagList")
+            ta_ob = Tag()
```

Comment:
@Etomovich, the variable name (`ta_ob`) is not descriptive enough, this can be misleading.

Reply:
Let me make it more descriptive